niworkflows/interfaces/nibabel.py

```diff
@@ -89,3 +89,68 @@ def _run_interface(self, runtime):
         maskimg.to_filename(self._results['out_mask'])
 
         return runtime
+
+
+class _FourToThreeInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='input 4d image')
+
+
+class _FourToThreeOutputSpec(TraitedSpec):
+    out_files = OutputMultiObject(File(exists=True),
+                                     desc='output list of 3d images')
+
+
+class FourToThree(SimpleInterface):
+    """Split a 4D dataset along the last dimension
+    into a series of 3D volumes."""
+
+    input_spec = _FourToThreeInputSpec
+    output_spec = _FourToThreeOutputSpec
+
+    def _run_interface(self, runtime):
+        filenii = nb.load(self.inputs.in_file)
+        if len(filenii.shape) != 4:
+            raise RuntimeError('Input image (%s) is not 4D.' % filenii)
+
+        files_3d = nb.four_to_three(filenii)
+        out_files = []
+        for i, file_3d in enumerate(files_3d):
+            out_file = fname_presuffix(in_file, suffix="_tmp_{}".format(i))
+            file_3d.to_filename(out_file)
+            out_files.append(out_file)
+
+        self._results['out_files'] = out_files
+        return runtime
+
+
+class _MergeSeriesInputSpec(BaseInterfaceInputSpec):
+    in_files = InputMultiObject(File(exists=True, mandatory=True,
+                                   desc='input list of 3d images'))
+
+
+class _MergeSeriesOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='output 4d image')
+
+
+class MergeSeries(SimpleInterface):
+    """Merge a series of 3D volumes along the last dimension into a single
+    4D image."""
+
+    input_spec = _MergeSeriesInputSpec
+    output_spec = _MergeSeriesOutputSpec
+
+    def _run_interface(self, runtime):
+        nii_list = []
+        for f in self.inputs.in_files:
+            filenii = nb.load(f)
+            filenii = nb.squeeze_image(filenii)
+            if len(filenii.shape) != 3:
+                raise RuntimeError('Input image (%s) is not 3D.' % f)
+            else:
+                nii_list.append(filenii)
+        img_4d = nb.funcs.concat_images(nii_list)
```

Comment:
```suggestion
        img_4d = nb.concat_images(nii_list)
```
is this right, @effigies?

Reply:
both seem to work here, so whatever @effigies prefers?